fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -609,7 +609,7 @@ void DLApi::init() {
 	                   headerVersion >= 0);
 	loadClientFunction(&api->transactionGetRange, lib, fdbCPath, "fdb_transaction_get_range", headerVersion >= 0);
 	loadClientFunction(
-	    &api->transactionGetMappedRange, lib, fdbCPath, "fdb_transaction_get_mapped_range", headerVersion >= 700);
+	    &api->transactionGetMappedRange, lib, fdbCPath, "fdb_transaction_get_mapped_range", headerVersion >= 710);
 	loadClientFunction(
```

Comment:
Also fdb_transaction_get_range_and_flat_map must be loaded, for headerVersion < 710

Reply:
Same. No longer support `fdb_transaction_get_range_and_flat_map` in 7.0.